aten/src/ATen/native/TensorProperties.cpp

```diff
@@ -13,6 +14,21 @@ bool is_same_size(const Tensor& self, const Tensor& other) {
   return self.sizes().equals(other.sizes());
 }
 
+bool nested_is_same_size(const Tensor& self, const Tensor& other) {
```

Comment:
Since we've made this nested specific, probably want to update this function to just expect both args to both be nested tensors and error otherwise